### ISSUE TITLE
[HOLD wave6] LOW - Ensure consistent use of `set`, `changed` and `removed` in request history

### ISSUE THREAD

**Issue by JmillsExpensify** [Expensify/App#24443]
We're adding these request history items for wave6, but we need to make sure that we're consistently applying history changes to all request data.

**Comment by luacmartins** [comment]
on hold, not overdue

**Comment by conorpendergrast** [comment]
@JmillsExpensify Eh, weird. I'm on parental leave and wanted to re-assign but this is only letting you be the assignee. If you're fine with that, great! If you need someone else assigned, maybe use Bug instead?

**Comment by JmillsExpensify** [comment]
I'm fine with that. No worries.

**Comment by JmillsExpensify** [comment]
Still waiting on wave6. I'll also add it to that project board.

**Comment by JmillsExpensify** [comment]
Still waiting on wave6

**Comment by JmillsExpensify** [comment]
Still pending wave6

**Comment by JmillsExpensify** [comment]
Same

**Comment by JmillsExpensify** [comment]
@sophiepintoraetz I wonder if we should go ahead and close this issue, given that it's ultimately now wrapped up in your system messages project?

**Comment by JmillsExpensify** [comment]
Discussed with Sophie offline and she agree that this is part of her wave8 project, so I'm closing this issue!
